rules/macos/execution_curl_spawned_from_installer_package.toml

```diff
@@ -0,0 +1,69 @@
+[metadata]
+creation_date = "2021/02/23"
+maturity = "production"
+updated_date = "2021/02/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when the built in macOS Installer program is used to install a package that spawns a shell and runs
+the cURL command. This activity has been observed being leveraged by malware.
+"""
+false_positives = [
+    "Custom organization-specific macOS packages that use .pkg files to run cURL could trigger this rule. If known behavior is causing false positives, it can be excluded from the rule.",
+]
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Installer Spawning cURL from macOS Package"
+references = ["https://redcanary.com/blog/clipping-silver-sparrows-wings"]
+risk_score = 47
+rule_id = "99239e7d-b0d4-46e3-8609-acafcf99f68c"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Execution"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.category:process and event.type:start and
+ process.parent.executable: "/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer" and
+ process.executable: (/bin/bash or /bin/zsh or /bin/sh) and
```

Comment:
Nice rule, @peasead. Have you considered including other shell processes here like dash, tcsh, csh, ksh, fish, or do you think that's excessive?

Reply:
I think those are probably good ideas.

So, thinking?
```
...
process.executable: (/bin/bash or /bin/zsh or /bin/sh or /bin/dash or \
 /bin/tcsh or /bin/csh or /bin/ksh or /bin/fish
...
```

Reply:
LGTM. Our users can always add exceptions to the rule to filter benign behavior. 👍

Reply:
Moved to EQL to detect any network events generated from a macOS Installer execution.